docs(android): Document removing sendOptionsToKeyboard API (#13220)
keymanapp/keyman: android/docs/engine/KMManager/setLongpressDelay.md

## Summary

> The `setLongpressDelay()` method sets the number of milliseconds to trigger a longpress gesture as a stored preference and sends it to the KeymanWeb Engine.

Reviewer: ```suggestion
The `setLongpressDelay()` method sets the number of milliseconds to trigger a longpress gesture, applies to the keyboard, and stores preference.
```

This is an internal implementation detail, and doesn't need to be in the API docs :grin:
